node/src/accountant/big_int_processing/big_int_db_processor.rs

```diff
@@ -407,6 +407,27 @@ impl StdNumParamFormNamed {
     }
 }
 
+pub struct Param<'a> {
+    value_pair: (&'a str, &'a dyn ExtendedParamsMarker),
+    participates_in_overflow_update_clauses: bool,
```

Comment:
It might be good to put the flag either at the beginning in both places or at the end in both places...although if you split the `filter_and_merge_pure_and_wei_params()` function into two, this is moot.